docs/api/browser-window.md

```diff
@@ -467,23 +467,23 @@ The `BrowserWindow` class has the following static methods:
 
 #### `BrowserWindow.getAllWindows()`
 
-Returns an array of all opened browser windows.
+Returns `BrowserWindow[]` - An array of all opened browser windows.
```

Comment:
I think we need to go back to #7337 if you disagree with the new format for representing these things.  It was approved by @zeke in that issue so perhaps you should bring that up with him before I start making these changes.

I think the key thing here is the difference between human readable documentation and clear / accurate / formatted documentation.  A think this a compromise between the two (both readable and follows an exact format).